thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/datalayer/dto/AlertTemplateDTO.java

```diff
@@ -23,6 +23,7 @@
 public class AlertTemplateDTO extends AbstractDTO {
 
   private String name;
+  private String namespace;
```

Comment:
This could be tricky. 

Either this should be at the top level like `AbstractDTO` since, potentially most objects will have some kind of auth.

Another question: Is `namespace` the only identifier and entity needed for auth in the foreseeable future? Else would suggest creating a container here instead of having it as `String`.

Reply:
We only need id/name, namespace, and entity type, and that's all for the foreseeable future. Namespace on AbstractDTO makes sense to me.